src/product/security.md

```diff
@@ -118,6 +118,11 @@ When running the FlowFuse Device Agent on a user's own hardware and network, it
  - `mqtt.flowfuse.com`
  - `registry.npmjs.org` - to enable installing of Node-RED modules
 
+When running the Node-RED editor on a user's own netwotk, it will require access to the following domains to be fully operational:
+
+- `app.flowfuse.com`
+- `catalogue.nodered.org`
+  
```

Comment:
```suggestion
When running the Node-RED editor on a user's own network, it will require access to the following domains to be fully operational:

- `app.flowfuse.com`
- `mqtt.flowfuse.com`
- `*.flowfuse.cloud`
- `catalogue.nodered.org`
  
```